examples/getstarted/config/locales/sk_sk.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not really needed as it is not used in the example app. Do you mind removing it?